---
layout: article
title: Instrument-Driven Workflows in BaseSpace
hide_welcome_banner: true
---
-----------------------
# Instrument-Driven Workflows in BaseSpace

**What are they?**

Instrument-Driven Workflows in BaseSpace are any analyses that are automatically run on your Runs after they are uploaded to BaseSpace, these are specified on the Sample Sheet on the machine before the Run is started.  Essentially, as soon as the Run is finished uploading, the user can choose to automatically run some standard workflows to get more data on their Samples.

The workflows are the same if BaseSpace is not used, so from the MiSeq instrument the user can choose to run these analyses locally on the machine without uploading to BaseSpace via the MiSeq Reporter software on the machine, but the data and workflows are the same between BaseSpace and the instrument.

**When are they used?**

These workflows are specified in the Sample Sheet before the Run is started, so the user must know what workflow they wish to run before sequencing begins.

These workflows are only available for the MiSeq instrument.  In the future, we hope to introduce more workflows that are driven from the various Illumina sequencing instruments.

## Resequencing

The sample sheet driven Resequencing app compares the DNA sequence in the samples against a reference genome and identifies any variants (SNPs or indels) relative to the reference sequence. The main output files generated by the Resequencing workflow are BAM files (containing the alignment results) and VCF files (containing the variant calls).

The Resequencing workflow can only be used to analyze MiSeq sequencing results.

The Resequencing App Results Page provides four graphs:

* Low Percentage Graph
* High Percentage Graph
* Clusters Graph
* Mismatch Graph

The Resequencing Sample Details Page provides five panes:

* Samples Table
* Coverage Graph
* Q-Score Graph
* Variant Score Graph
* Variants Table

The graphs and variants table display data for the chromosome that is selected in the drop-down list.

## Custom/PCR Amplicon

The Custom/PCR Amplicon workflow evaluates short regions of amplified DNA (amplicons) for variants. The focused sequencing of amplicons enables high-coverage sequencing of particular regions across many samples. The main output files generated by the Custom/PCR Amplicon workflow are BAM files (containing the aligned reads) and VCF files (containing the variant calls). The Custom/PCR Amplicon workflow supports multiple manifests (containing the probe regions) and consensus sequence reporting for multi-manifest runs.

The Custom/PCR Amplicon workflow can only be used to analyze MiSeq sequencing results

The Custom/PCR Amplicon App Results Page provides a four graphs:

* Low Percentage Graph
* High Percentage Graph
* Clusters Graph
* Mismatch Graph

The PCR amplicon Sample Details page provides six panes:

* Samples Table
* Amplicons Table
* Coverage Graph
* Q-Score Graph
* Variant Score Graph
* Variants Table

The graphs and variants table display data for the amplicon that is selected in the Amplicon Table.

## Library QC

The Library QC workflow is intended for evaluating the abundance, fragment length, and sample quality of libraries. The analysis performed in the Library QC workflow is similar to the Resequencing workflow. The Library QC workflow does not perform variant calling; instead, it provides a report of the characteristics of each sample.

The Library QC workflow can only be used to analyze MiSeq sequencing results

The Library QC App Results page provides a four graphs:

* Low Percentage Graph
* High Percentage Graph
* Clusters Graph
* Mismatch Graph

The Library QC Sample Details Page provides four panes:

* Samples Table
* Coverage Graph
* Q-Score Graph
* Sample QC Table

The graphs display data for the chromosome that is selected in the drop-down list.

## Small RNA Analysis

The Small RNA workflow measures the abundance of various types of short RNA sequences, particularly miRNA. It is suitable for identifying and quantifying miRNA expression and for comparing abundance across samples.

The Small RNA workflow can only be used to analyze MiSeq sequencing results

The small RNA analysis App Results page provides access to two graphs.

* Clusters Graph
* Trimmed Lengths

The Small RNA Sample Details Page provides three panes:

* Small RNA Samples Table
* Small RNA Pie Chart
* Small RNA Graph

## Metagenomics Analysis

The Metagenomics workflow enables the analysis of 16S ribosomal RNA, a component of the 30S subunit of prokaryotic ribosomes. The 16S ribosomal sequences from an environmental sample can be analyzed to determine which organisms are present. In MiSeq Reporter, a naïve Bayesian classifier (based on [Wang et al., Appl. Environ. Microbiol. (2007) Aug; 73(16):5261-7](http://www.ncbi.nlm.nih.gov/pubmed/17586664?dopt=AbstractPlus)) has been implemented that has been optimized for Illumina paired-end reads. Our 16S rRNA data store contains sequences from the May 2011 release of the [Greengenes 16S rRNA database](http://greengenes.lbl.gov/). The main output of this workflow is a classification of reads at several taxonomic levels (kingdom, phylum, class, order, family, genus).

The Metagenomics workflow can only be used to analyze MiSeq sequencing results

The metagenomics App Results page provides one graph.

* Clusters Graph

The Metagenomics Sample Details Page provides two panes:

* Samples Table
* Metagenomics Pie Chart

## De Novo Assembly

The Assembly workflow enables de novo assembly of a draft genome directly from the sequencing reads. Because assembly relies upon significant coverage of the genome, this workflow is best suited for the assembly of small genomes (up to 5 to 10 MB). The assembly process uses the Velvet software (Velvet: algorithms for de novo short read assembly using de Bruijn graphs (2008) D.R. Zerbino and E. Birney. Genome Research 18:821–829).

The Assembly workflow can only be used to analyze MiSeq sequencing results

The de novo assembly App Results page provides access to three graphs:

* Low Percentage Graph
* High Percentage Graph
* Clusters Graph

The De Novo Assembly Sample Details Page provides two panes:

* De Novo Assembly Samples Table
* Samples Graph

## Generate FASTQ

The app Generate FASTQ does not perform any analysis, but generates FASTQ files for download and shows basic summary data. The Generate FASTQ app can be used with all sequencing instruments that BaseSpace supports. For more information, see FASTQ Files.

Generate FASTQ is also used to analyze RNA-Seq samples from MiSeq.
